modules/store/src/selector.ts

```diff
@@ -122,34 +122,19 @@ export function defaultMemoize(
   return { memoized, reset, setResult, clearResult };
 }
 
-export function createSelector<State, S1, Result>(
-  s1: Selector<State, S1>,
-  projector: (s1: S1) => Result
+export function createSelector<State, S extends unknown[], Result>(
```

Comment:
It would be great to add a full name for the `S` generic because it's not clear at first what is its purpose. The name could be: `Slices`